features/verblijfstitelhistorie.feature

```diff
@@ -1,149 +1,333 @@
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding, datumEinde, datumIngang |
+    | peildatum           | 2019-09-28            |
+    Dan heeft de response geen verblijfstitelhistorie
 
-  De verblijfstitels worden in het antwoord aflopend gesorteerd op ingangsdatum, zodat de meest actuele bovenaan staat.
-  Als er een actuele verblijfstitel is (categorie 10) staat deze als eerste verblijfstitel in het antwoord.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met peildatum voor de ingangsdatum van de eerste verblijfstitel
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding, datumEinde, datumIngang |
+    | peildatum           | 2019-09-28            |
+    Dan heeft de response geen verblijfstitelhistorie
 
-  Als een verblijfstitel, actueel of historisch, in onderzoek is, en dit onderzoek is niet afgerond (Datum einde onderzoek is leeg), wordt inOnderzoek gevuld voor betreffende verblijfstitel.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met peildatum gelijk aan de datum einde van een verblijfstitel
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding, datumEinde, datumIngang |
+    | peildatum           | 2019-09-01            |
+    Dan heeft de response geen verblijfstitelhistorie
 
-  Als wel het begin van de periode (datumVan) wordt opgegeven, maar geen einde van de periode (datumTotEnMet), dan worden alle verblijfstitels vanaf de datumVan in het antwoord opgenomen.
-  Als wel het einde van de periode (datumTotEnMet) wordt opgegeven, maar geen begin van de periode (datumVan), dan worden alle verblijfstitels tot en met de datumTotEnMet in het antwoord opgenomen.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met peildatum gelijk aan de datum ingang van een verblijfstitel
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding.code, datumEinde, datumIngang |
+    | peildatum           | 20160901              |
+    Dan heeft de response een verblijfstitelhistorie met de waarden
+    | aanduiding.code | datumIngang | datumEinde |
```

Comment:
Zelfde structuur als in andere features gebruiken?

```
      Dan heeft de response een verblijfstitel met de volgende gegevens
      | naam                          | waarde   |
      ...
```

Reply:
of is dat onhandig bij het controleren van meerdere verblijfstitels in de response?